lib/auth/accountrecovery.go

```diff
@@ -351,6 +354,15 @@ func (a *Server) CreateAccountRecoveryCodes(ctx context.Context, req *proto.Crea
 		return nil, trace.AccessDenied(unableToCreateCodesMsg)
 	}
 
+	// Verify if the user is local.
+	switch user, err := a.GetUser(ctx, token.GetUser(), false /* withSecrets */); {
+	case err != nil:
+		// err swallowed on purpose.
+		return nil, trace.AccessDenied(unableToCreateCodesMsg)
```

Comment:
I've opted to be opaque here, following the surrounding code, but didn't feel the same need below.